app/utils/CheckYourAnswersHelper.scala

```diff
@@ -50,6 +50,15 @@ class CheckYourAnswersHelper @Inject()(countryOptions: CountryOptions)(userAnswe
       )
   }
 
+  def trusteesUkAddress(index: Int): Option[AnswerRow] = userAnswers.get(TrusteesUkAddressPage(index)) map {
+    x =>
+      AnswerRow(
+        "trusteesUkAddress.checkYourAnswersLabel",
+        ukAddress(x),
+        routes.TrusteesUkAddressController.onPageLoad(CheckMode, index).url
+      )
```

Comment:
Do you need to pass through trusteeName as an argument?

Reply:
Done